dhcpv4/server4/server_test.go

```diff
@@ -5,23 +5,15 @@ package server4
 import (
 	"context"
 	"log"
-	"math/rand"
 	"net"
 	"testing"
-	"time"
 
 	"github.com/insomniacslk/dhcp/dhcpv4"
 	"github.com/insomniacslk/dhcp/dhcpv4/nclient4"
 	"github.com/insomniacslk/dhcp/interfaces"
 	"github.com/stretchr/testify/require"
 )
 
-func init() {
```

Comment:
Is this part of this PR?

Reply:
Yeah, lint was complaining that this function is deprecated:

https://pkg.go.dev/math/rand#Seed